roles/host-provision/tasks/config-tasks.yml

```diff
@@ -17,10 +17,46 @@
   ping:
   tags: host-provision
 
-# Placeholder for the core config tasks to be executed by the host-provision.sh utility
-# - name: config-tasks | Create LVM layer
-# ...
 
+# Not including ntp set up, as we already have code path built into `base-provision` role
+# that processes the commmand line switch: `--ntp-pref`
 
-# - name: config-tasks | For RHEL, subscription manager
-# ...
+
+# LVM creation
+# The net result will be that `/dev/mapper/db-sw` can be input into `install-oracle.sh` for local u01 storage
+- name: config-tasks | Create LVM layer
```

Comment:
Using LVM for the Oracle software volume was originally intended as a workaround for the toolkit's inability to use different LUN names on cluster nodes.  I'm not sure it's usable as a blanket solution though;  for example, we support ()and, via example, encourage) having multiple LUNs to store content like the Oracle diag_dest.  Also, we currently have disk mounts and purposes uncoupled for flexibility, and I think this change would make that more difficult.

So bringing back an old idea, might it be cleaner to add support for host-specific mountpoint names in the [data mount configuration file](https://github.com/google/bms-toolkit/blob/master/docs/user-guide.md#data-mount-configuration-file) instead, avoiding the need for this configuration entirely?

Reply:
I still think LVM is a lot of complexity for a relatively simple issue of when a cluster has different WWNs for private volumes.  But as you say it does work in this configuration, and we're not entirely closing the door to refactoring in the future.